packages/mask/background/database/persona/app.ts

```diff
@@ -0,0 +1,192 @@
+import type {
+    PersonaRecord,
+    FullPersonaDBTransaction,
+    PersonasTransaction,
+    ProfileRecordDB,
+    PersonaRecordWithPrivateKey,
+    ProfileRecord,
+    ProfileTransaction,
+    RelationRecord,
+    RelationTransaction,
+} from './type'
+import type { ProfileIdentifier, PersonaIdentifier } from '@masknet/shared-base'
+
+export async function createPersonaDB(record: PersonaRecord): Promise<void> {
+    return
+}
+
+export async function createPersonaByProfileDB(
+    query: ProfileIdentifier,
+    t?: FullPersonaDBTransaction<'readonly'>,
```

Comment:
you shouldn't ask for a transaction in the mobile implementation

Reply:
To minimize code changes, the transaction is not passed in the mobile environment, and the parameter is changed to optional